docs/api_v2/api_v2_reference.apib

```diff
@@ -2918,238 +3123,413 @@ The user account is identified by the provided authentication token.
 - status:           `ABORTED`                  (string, required)
 - meta                                         (object, required)
-    - source:        `WEB`                     (string, required)
+    - origin:        `WEB`                     (string, required)
     - clientIp:      `172.234.12.34`           (string, required)
     - userAgent:     `Mozilla/5.0 (iPad)`      (string, required)
 - stats                                        (object, nullable)
     - workersUsed:   1                         (number, required)
     - durationMillis 1000                      (number, required)
-- buildNumber:       `0.0.1`                   (string, required)
-- readme:            `# Magic actor\nThis actor is magic.`            (string, nullable)
+    - runTimeSecs:   5.718                     (number, required)
+    - readyTimeSecs: 0.035                     (number, required)
+    - initTotalSecs: 0.139                     (number, required)
+    - emfileError                              (object, nullable)
+- options                                      (object, nullable)
+    - useCache:      false                     (boolean, nullable)
+    - memoryMbytes:  1024                      (number, nullable)
+    - diskMbytes:    2048                      (number, nullable)
 
 ## Run (object)
 - id:               `HG7ML7M8z78YcAPEB`        (string, required)
-- buildId:          `HG7ML7M8z78YcAPEB`        (string, required)
 - actId:            `HDSasDasz78YcAPEB`        (string, required)
-- actTaskId:        `KJHSKHausidyaJKHs`        (string, nullable)
-- startedAt:        `2015-11-30T07:34:24.202Z` (string, required)
-- finishedAt:       `2015-12-12T09:30:12.202Z` (string, required)
+- userId:           `7sT5jcggjjA9fNcxF`        (string, required)
+- actorTaskId:      `KJHSKHausidyaJKHs`        (string, nullable)
+- startedAt:        `2019-11-30T07:34:24.202Z` (string, required)
+- finishedAt:       `2019-12-12T09:30:12.202Z` (string, required)
 - status:           `RUNNING`                  (string, required)
-- defaultKeyValueStoreId: `eJNzqsbPiopwJcgGQ`  (string, required)
-- defaultDatasetId: `wmKPijuyDnPZAPRMk`        (string, required)
-- defaultRequestQueueId: `FL35cSF7jrxr3BY39`   (string, required)
 - meta                                         (object, required)
-    - source:       `WEB`                      (string, required)
-    - clientIp:     `172.234.12.34`            (string, required)
+    - origin:       `WEB`                      (string, required)
+    - clientIp:      `172.234.12.34`           (string, required)
     - userAgent:    `Mozilla/5.0 (iPad)`       (string, required)
 - stats                                        (object, required)
-    - workersUsed:    1                        (number, required)
-    - resurrectCount: 2                        (number, required)
+    - inputBodyLen:             240            (number, nullable) # TODO: review 'nullable' on whole segment
```

Comment:
And same here @jancurn  I'd keep:

- restartCount
- resurrectCount
- mem[Avg|Max|Current]Bytes
- cpu[Avg|Max|Current]Usage
- netRxBytes
- netTxBytes
- durationMillis
- metamorph
- computeUnits

and remove:

- workersUsed
- initPrepSecs
- initImagePullSecs
- initContainerCreateSecs
- initContainerStartSecs
- runTimeSecs
- readyTimeSecs
- initTotalSecs
- emfileError

Reply:
Makes sense